extensions/wikia/EditPageLayout/EditPageLayoutHelper.class.php

```diff
@@ -253,7 +252,7 @@ public function showMobilePreview( Title $title ) {
 	 * @return bool
 	 */
 	public static function isCodePageWithPreview( Title $title ) {
-		return self::isInfoboxTemplate( $title );
+		return $title->getNamespace() === NS_TEMPLATE;
```

Comment:
[`inNamespace`](https://github.com/Wikia/app/blob/ef1e2e3331c6bf/includes/Title.php#L947) is the preferred method for checking if a title is in a namespace.